Haruby-iOS/Sources/Feature/Input/InputViewController.swift

```diff
@@ -0,0 +1,319 @@
+//
+//  InputViewController.swift
+//  Haruby-iOS
+//
+//  Created by Seo-Jooyoung on 10/10/24.
+//
+
+import UIKit
+import ReactorKit
+import RxSwift
+import RxCocoa
+import SnapKit
+
+class InputViewController: UIViewController, View {
+    // MARK: - Properties
+    var disposeBag = DisposeBag()
+    typealias Reactor = InputViewReactor
+    
+    let segmentedControl: UISegmentedControl = {
+        let control = UISegmentedControl(items: ["지출", "수입"])
```

Comment:
SegmentedControl에 사용될 items 요소들이 여러 곳에서 사용되고 있기 때문에 Constant로 관리해주는 것이 좋아보입니다.
```suggestion
enum TransactionType: Int {
    case expense
    case income
    
    var text: String {
        switch self {
        case .expense:
            return "지출"
        case .income:
            return "수입"
        }
    }
}

let control = UISegmentedControl(items: [
    TransactionType.expense.text, 
    TransactionType.income.text
])
```